clone/cmd/sumdbverify/verify.go

```diff
@@ -76,30 +78,53 @@ func verifyLeaves(ctx context.Context, db dataSource) (uint64, error) {
 		// golang.org/x/text v0.3.0 h1:g61tztE5qeGQ89tm6NTjjM9VPIm088od1l6aSorWRWg=
 		// golang.org/x/text v0.3.0/go.mod h1:NqM8EUOU14njkJ3fqMW+pc6Ldnwhi/IjpwHt7yyuwOQ=
 		//
+		line0RE := regexp.MustCompile(`(.*) (.*) h1:(.*)`)
```

Comment:
nit: I think these could be moved outside of the loop to avoid recompiling for each leaf?

Reply:
Done.